src/main/java/io/jenkins/plugins/luxair/ImageTagParameterDefinition.java

```diff
@@ -38,24 +38,26 @@
     private final String credentialId;
     private String defaultTag;
     private Ordering tagOrder;
+    private Boolean verifySsl;
     private String errorMsg = "";
 
     @DataBoundConstructor
     @SuppressWarnings("unused")
     public ImageTagParameterDefinition(String name, String description, String image, String filter,
-                                       String registry, String credentialId) {
-        this(name, description, image, filter, "", registry, credentialId, config.getDefaultTagOrdering());
+                                       String registry, String credentialId, Boolean verifySsl) {
+        this(name, description, image, filter, "", registry, credentialId, config.getDefaultTagOrdering(), verifySsl);
```

Comment:
```suggestion
                                       String registry, String credentialId) {
        this(name, description, image, filter, "", registry, credentialId, config.getDefaultTagOrdering(), config.getDefaultVerifySsl());
```
This would be what I would change it to, so that ppl who use the JobDsl won't be required to add the parameter.
> Note: If you use the JobDsl you can still add the parameter and therefor set the property, but it is optional instead of required.

Reply:
Thanks for the feedback! That seems very sensible, will commit